website/source/docs/commands/connect/envoy.html.md.erb

```diff
@@ -94,6 +79,54 @@ allowed to access by [Connect intentions](/docs/connect/intentions.html).
    always specifies `--config-file` and `--v2-config-only` and by default passes
    `--disable-hot-restart` see [hot restart](#hot-restart).
 
+#### Envoy Sidecar Proxy Options
+
+* `-sidecar-for` - The _ID_ (not name if they differ) of the service instance
+  this proxy will represent. The target service doesn't need to exist on the
+  local agent yet but a [sidecar proxy
+  registration](/docs/connect/registration/service-registration.html) with
+  `proxy.destination_service_id` equal to the passed value must be present. If
+  multiple proxy registrations targeting the same local service instance are
+  present the command will error and `-proxy-id` should be used instead.
+
+-> **Note:** If ACLs are enabled, a token granting `service:write` for the
+  _target_ service (configured in `proxy.destination_service_name`) must be
+  passed using the `-token` option or `CONSUL_HTTP_TOKEN` environment variable.
+  This token authorizes the proxy to obtain TLS certificates representing the
+  target service.
+
+#### Envoy Mesh Gateway Options
+
+* `-mesh-gateway` - Flag to indicate that Envoy should be configured as a Mesh
+  Gateway. If multiple mesh gateways are managed by the same local agent then
+  `-proxy-id` should be used as well to specify the instance this represents.
+
+* `-register` - Indicates that the mesh gateway service should be registered
+  with the local agent instead of expecting it to already exist. This flag
+  is unused for traditional sidecar proxies.
+
+* `-address` - The address to advertise for services within the local datacenter
+  to use to reach the mesh gateway instance. This flag is used in combination with
+  `-register`. This takes the form of `<ip address>:<port>` but also supports go-sockaddr
+  templates.
+
+* `-wan-address` - The address to advertise for services within remote datacenters
+  to use to reach the mesh gateway instance. This flag is used in combination with
+  `-register`. This takes the form of `<ip address>:<port>` but also supports go-sockaddr
+  templates.
+
+* `-service` - The name of the mesh gateway service to register. This flag is used
+  in combination with `-register`.
+
+* `-deregister-after-critical` - The amount of time the gateway services health check can
+  be failing before being deregistered. This flag is used in combination with `-register`
+
+-> **Note:** If ACLs are enabled, a token granting `service:write` for the
```

Comment:
I think if you tab this in it will nest under the bullet that it refers to.

Reply:
This one is referring to this sub-section in general though, not necessarily the preceding option.